app/uploaders/avatar_uploader.rb

```diff
@@ -3,6 +3,15 @@
 class AvatarUploader < CarrierWave::Uploader::Base
   include Cloudinary::CarrierWave
 
+  # Sets the user's avatar to the given image url if the user
+  # doesn't currently have an avatar.
+  def self.set_avatar_from_url!(user, url)
+    if !user.no_third_party_avatar && !user.avatar.file
```

Comment:
To address `Use a guard clause instead of wrapping the code inside a conditional expression` from Code Climate, I think we can do the following:

```
def self.set_avatar_from_url!(user, url)
  return unless !user.no_third_party_avatar && !user.avatar.file
  
  user.remote_avatar_url = url
  user.save
end
```